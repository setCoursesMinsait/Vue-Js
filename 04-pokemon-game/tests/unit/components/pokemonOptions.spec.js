import { shallowMount } from "@vue/test-utils";
import PokemonOptions from "@/components/PokemonOptions"

import { pokemons } from "../mocks/pokemons.mock";

describe( 'Pokemon Option Components', () => {

  let wrapper = shallowMount( PokemonOptions )

  beforeEach(() => {
    wrapper = shallowMount( PokemonOptions, {
      props: {
        pokemons
      }
    })
  })

  test( 'debe de hacer match con en el snapshot', () => {
    expect( wrapper.html() ).toMatchSnapshot()
  })
})