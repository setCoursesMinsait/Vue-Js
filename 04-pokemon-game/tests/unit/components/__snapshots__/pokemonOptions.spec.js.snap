// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pokemon Option Components debe de hacer match con en el snapshot 1`] = `
<div class="options-container">
  <ul>
    <li>pikachu</li>
    <li>pikachu</li>
    <li>pikachu</li>
    <li>pikachu</li>
    <li>pikachu</li>
  </ul>
</div>
`;
